### PR TITLE
Rename runner to trainer

### PR DESCRIPTION
**Pull request by zequnyu**
https://github.com/rlworkgroup/garage/issues/1048.

### CODE REVIEW

File: src/garage/trainer/trainer.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def __init__(self, total_epoch, total_itr, total_env_steps, last_episode):
 
 class SetupArgs:
     # pylint: disable=too-few-public-methods
-    """Arguments to setup a runner.
+    """Arguments to setup a trainer.
```
Comment (ryanjulian) [inline src/garage/trainer/trainer.py:38]: unless we feel that we are going to have a lot of these, perhaps we can just put this in `src/garage/trainer.py` (and also `TFTrainer` there too)

File: examples/tf/trpo_cartpole_recurrent.py [hunk 2]
```diff
@@ -43,8 +43,13 @@ def trpo_cartpole_recurrent(ctxt, seed, n_epochs, batch_size, plot):
 
     """
     set_seed(seed)
+<<<<<<< HEAD
     with LocalTFRunner(snapshot_config=ctxt) as runner:
         env = GymEnv('CartPole-v1', max_episode_length=100)
+=======
+    with TFTrainer(snapshot_config=ctxt) as trainer:
+        env = GymEnv('CartPole-v1')
+>>>>>>> Rename runner to trainer
```
Comment (maliesa96) [inline examples/tf/trpo_cartpole_recurrent.py:53]: Looks like you have some leftover text from git

File: examples/tf/trpo_cubecrash.py [hunk 3]
```diff
@@ -23,17 +23,22 @@ def trpo_cubecrash(ctxt=None, seed=1, max_episode_length=5, batch_size=4000):
...
     """
     set_seed(seed)
+<<<<<<< HEAD
     with LocalTFRunner(ctxt) as runner:
         env = normalize(
             GymEnv('CubeCrash-v0', max_episode_length=max_episode_length))
+=======
+    with TFTrainer(ctxt) as trainer:
+        env = normalize(GymEnv('CubeCrash-v0'))
+>>>>>>> Rename runner to trainer
```
Comment (maliesa96) [inline examples/tf/trpo_cubecrash.py:41]: here

File: examples/torch/maml_ppo_half_cheetah_dir.py [hunk 4]
```diff
@@ -72,9 +73,15 @@ def maml_ppo_half_cheetah_dir(ctxt, seed, epochs, episodes_per_task,
...
 
+<<<<<<< HEAD
     runner.setup(algo, env)
     runner.train(n_epochs=epochs,
                  batch_size=episodes_per_task * env.spec.max_episode_length)
+=======
+    trainer.setup(algo, env)
+    trainer.train(n_epochs=epochs,
+                 batch_size=episodes_per_task * max_episode_length)
+>>>>>>> Rename runner to trainer
```
Comment (maliesa96) [inline examples/torch/maml_ppo_half_cheetah_dir.py:84]: here

**Review by ryanjulian — APPROVED** [review]

**Review by maliesa96 — APPROVED** [review]

**Review by irisliucy — APPROVED** [review]

**Pull request merged**